packages/roosterjs-editor-plugins/lib/TabIndent/TabIndent.ts

```diff
@@ -83,6 +87,33 @@ export default class TabIndent implements EditorPlugin {
                     }
                 }
             }
+        } else {
+            let nodeAtCursor = getNodeAtCursor(this.editor);
+            let [isBlockQuoteEvent, blockQuoteElement] = this.isBlockQuoteEvent(event, [KEY_ENTER], nodeAtCursor);
+            if (isBlockQuoteEvent) {
+                let keybordEvent = (event as PluginDomEvent).rawEvent as KeyboardEvent;
```

Comment:
Both if and else block use the keyboardEvent, you could move it out to the beginning of this method.

Reply:
Changed